sensorprocess/sensorprocess.go

```diff
@@ -20,15 +20,11 @@ type Config struct {
 	CartoFacade            cartofacade.Interface
 	sensorProcessStartTime time.Time
 
-	Lidar                sensors.TimedLidarSensor
-	LidarName            string
-	LidarDataFrequencyHz int
-	currentLidarData     LidarData
+	Lidar            sensors.TimedLidarSensor
+	currentLidarData LidarData
 
-	IMU                sensors.TimedIMUSensor
-	IMUName            string
-	IMUDataFrequencyHz int
-	currentIMUData     IMUData
```

Comment:
Refactoring: `Lidar` contains already the lidar name and data frequency information.